pkg/kubelet/cadvisor/helpers_linux.go

```diff
@@ -26,24 +26,40 @@ import (
 	cadvisorfs "github.com/google/cadvisor/fs"
 )
 
+// Once CAdvisor upstream changes are merged, we should remove this constant
+const LabelCrioContainers string = "crio-containers"
+
 // imageFsInfoProvider knows how to translate the configured runtime
 // to its file system label for images.
 type imageFsInfoProvider struct {
 	runtimeEndpoint string
 }
 
+// This is a temporary workaround to get stats for cri-o from cadvisor
+// and should be removed.
+// Related to https://github.com/kubernetes/kubernetes/issues/51798
+func detectCrioWorkaround(i *imageFsInfoProvider) bool {
```

Comment:
nit: I prefer functions to be defined below where they're called so you can read top to bottom

Reply:
done.